sandboxes/dynamicanalysis/Dockerfile

```diff
@@ -147,6 +147,9 @@ ENV NODE_PATH="/app/node_modules"
 # Test stuff
 RUN ruby --version && php --version && python3 --version && pip --version && node --version && npm --version && rustc --version && cargo --version
 
+ENV REQUESTS_CA_BUNDLE=/usr/local/share/ca-certificates/ca.crt
+ENV SSL_CERT_FILE=/usr/local/share/ca-certificates/ca.crt
```

Comment:
These env vars need to be copied into all the other dynamic analysis sandboxes, unless we're going to switch over to using this one. Thoughts @calebbrown?

Reply:
Yes, until they have been removed they should be replicated across the other sandbox images.

For reference: without these env vars things start to break due to cert errors.